### PR TITLE
Update Credits

### PR DESCRIPTION
**Pull request by AidenJamesYt**
added myself to the credits list, as an open source contributor.

## Notice
* Make sure you have read https://github.com/Epix-Incorporated/Adonis/blob/master/CONTRIBUTING.md before contributing!
* Ensure you title your PR correctly and that you include your proof of functionality as well as a description of what you have done, and a rationale as to why.
* **Remember to delete this section when you submit your PR!**

### CODE REVIEW

**Comment by PurpleCreativity** [comment]
A maintainer should add you to the credits list, also I can't find any merged PRs you have, what did you contribute?

**Comment by Dimenpsyonal** [comment]
You haven’t made any contribution though, so I can’t add you to the credits list

File: MainModule/Server/Shared/Credits.luau [hunk 1]
```diff
@@ -98,6 +98,7 @@ return {
 		{Text = "@PurpleCreativity",Desc = "Open Source Contributor"};
 		{Text = "@ScriptedConnor",	Desc = "Open Source Contributor"};
 		{Text = "@omwot",			Desc = "Open Source Contributor"};
+		{Text = "@AidenJamesYt",    Desc = "Open Source Contributor");
```
Comment (SuperCater) [inline MainModule/Server/Shared/Credits.luau:101]: Correction

```suggestion
		{Text = "@AidenJamesYt",    Desc = "Emotional Support");
```

**Review by SuperCater — CHANGES_REQUESTED** [review]

**Comment by AidenJamesYt** [comment]
i didnt mean to send it yet lmao

**Pull request closed without merging**